celer/homotopy.py

```diff
@@ -201,6 +204,7 @@ def celer_path(X, y, pb, eps=1e-3, n_alphas=100, alphas=None,
                                  positive) / 2
         elif pb == GRPLASSO:
             # TODO compute it with dscal to handle centering sparse
+            # TODO consider weights when computing alpha_max
```

Comment:
done?

Reply:
not yet, 
if I m not mistaken, I should divide by weight

```python
elif pb == GRPLASSO:
            # TODO compute it with dscal to handle centering sparse
            # TODO consider weights when computing alpha_max
            alpha_max = 0
            for g in range(n_groups):
                X_g = X[:, grp_indices[grp_ptr[g]:grp_ptr[g + 1]]]
                alpha_max = max(alpha_max, norm(X_g.T @ y / weights[g], ord=2))
            alpha_max /= n_samples
```

Reply:
yes